yourtrip/src/main/java/backend/yourtrip/domain/feed/service/FeedServiceImpl.java

```
@@ -175,6 +206,37 @@ public FeedUpdateResponse updateFeed(Long feedId, FeedUpdateRequest request) {
             }
         }
 
+        if (mediaFiles != null) {
+            // 기존 미디어 삭제 (S3에서도 삭제)
+            for (FeedMedia media : feed.getMediaList()) {
+                try {
+                    s3Service.deleteFile(media.getMediaS3Key());
+                } catch (Exception e) {
+                    // 로그 남기고 계속 진행
+                }
+            }
```

기존 S3 파일 삭제 시 try–catch 블록에서 예외를 처리하긴 하지만 현재는 catch 블록이 비어 있어 실제 삭제 실패 시 어떤 오류가 발생했는지 확인하기 어려워여
silent fail 형태라 문제 추적이 힘들어질 수 있어서 최소한 warn 로그라도 남겨주면 디버깅과 모니터링에 도움이 될 것 같아요 :)

---

warn 로그 남겼습니다!